src/index.njk

```diff
@@ -3,102 +3,172 @@ layout: default
-                <a class="mt-4 ff-btn ff-btn--primary uppercase inline-block" href="{{ site.appURL }}/account/create" onclick="capture('cta-get-started-today', {'position': 'secondary'})">Join Now</a>   
+                <a class="mt-4 ff-btn ff-btn--primary-outlined uppercase inline-block" href="{{ site.appURL }}/account/create" onclick="capture('cta-try-it-now', {'position': 'secondary'})">Try it out</a>   
             </div>
         </div>
     </div>
+    </div> 
+</div>
+<div class="w-full bg-white text-gray-50 -mb-1">
+    {% include "components/dividers/divider-flow--top-grey.svg" %}
 </div>
+<!-- FF Content -->
+<div class="ff-bg-light w-full pb-20 md:px-0">
+    <div class="container m-auto max-w-5xl pt-6 md:pt-3 px-6">
+        <h3 class="text-center w-full font-medium">Explore more about FlowFuse and Node-RED</h3>
+        <div class="w-full max-w-md md:max-w-none mx-auto flex flex-col md:grid md:grid-cols-3 gap-6 pt-10">
+            {%- for item in collections.posts | sort(attribute='item.date') | reverse | limit(1) -%}
+            <div class="w-full my-2 border rounded-lg hover:drop-shadow-lg hover:border-red-600 transition ease-in-out duration-300 bg-white">
+                <a href="{{ item.url }}" class="w-full flex flex-col group hover:no-underline">
+                    <div class="">
+                            {% if item.data.image %}
+                            <div class="relative border-b">
+                                <div class="w-full ff-image-cover ff-image-top-rounded">
+                                {% set imageSrc = ["./", item.data.image ] | join %}
+                                {% set imageDescription = ["Image representing ", item.data.title] | join %}
+                                {% image imageSrc, imageDescription, [285] %}
+                            </div>
+                            {% else %}
+                            {{ item.url | generatePostSVG |safe}}
+                            {% endif %}
+                        </div>
+                        <div class="p-5 pt-0">
+                        <h5 class="mt-4 mb-0 font-normal text-lg"><span class="text-gray-500 group-hover:text-red-600">{{ item.data.title }}</span></h5>
+                        <div class="text-sm py-1 text-gray-500 font-light">
+                            {{ item.data.description | excerpt | safe }}
+                        </div>
+                        </div>
+                    </div>
+                </a>
+            </div>
+            {%- endfor -%}
+            {%- for item in collections.webinar | sort(attribute='item.date') | reverse | limit(1) -%}
+            <div class="w-full my-2 border rounded-lg hover:drop-shadow-lg hover:border-red-600 transition ease-in-out duration-300 bg-white">
+                <a href="{{ item.url }}" class="w-full flex flex-col group hover:no-underline">
+                        <div class="">
+                            {% if item.data.image %}
```

Comment:
AFAIK there's not a webinar the else clause is triggered for? Or is this a copy + paste? If that is so, it shouldn've been a partial or shortcode?

Reply:
Ha, it's actually the same as above. For now this is fine. For next time; let's build an array with 2 elements and loop through them. :)